File: apps/docs/src/code/js/basics/process_env_vars.js
import { Sandbox } from '@e2b/sdk'

const sandbox = await Sandbox.create({
  id: 'base',
  envVars: {FOO: 'Hello'},
})

const proc = await sandbox.process.start({
  cmd: 'echo $FOO $BAR!',
  envVars: {BAR: 'World'}, // $HighlightLine
})
await proc.wait()
console.log(proc.output.stdout)
// output: Hello World!

await sandbox.close()

File: apps/docs/src/code/python/basics/process_env_vars.py
from e2b import Sandbox

session = Sandbox.create(
    id="Nodejs",
    env_vars={"FOO": "Hello"}
)

proc = session.process.start(
    "echo $FOO $BAR!",
    env_vars={"BAR": "World"},  # $HighlightLine
)
proc.wait()

print(proc.output.stdout)
# output: Hello World!

session.close()